rke/k8s_rke_system_images.go

```diff
@@ -2634,6 +2742,41 @@ func loadK8sRKESystemImages() map[string]v3.RKESystemImages {
 			WindowsPodInfraContainer:  m("rancher/kubelet-pause:v0.1.3"),
 			Nodelocal:                 m("k8s.gcr.io/k8s-dns-node-cache:1.15.7"),
 		},
+		// Enabled in Rancher v2.4.4
+		// Reminder: This template contains Nodelocal image which isn't in templates before v2.3.6
+		"v1.18.2-rancher1-2": {
```

Comment:
Since v1.18.2-rancher1-1 is brand-new to v2.4.4, you don't need a new template. You just need to update the nginx ingress image in the v1.18.2-rancher1-1 template.

Reply:
Ok, I'll do that